Update the document to show how to uninstall thin-edge.io (#1331)
thin-edge/thin-edge.io: docs/src/howto-guides/002_installation.md

dpkg -i tedge_mapper_0.5.0_armhf.deb
```

## Uninstall `thin-edge.io`

> The `thin-edge.io` can be uninstalled using a script, the script can be downloaded

Reviewer: ```suggestion
The `thin-edge.io` can be uninstalled using a script, that can be downloaded
```

Author: fixed

from below mentioned location. 

```shell

> https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/uninstall-thin-edge_io.sh

Reviewer: Shouldn't this be a `curl` or `wget` command?

Author: Yup, fixed it.

```
The uninstall script provides options as shown below.

```shell
USAGE:
    delete-thin-edge_io [COMMAND]
    
COMMANDS:
    remove     Uninstall thin-edge.io with keeping configuration files
    purge      Uninstall thin-edge.io and also remove configuration files
```

> > Note: The uninstall script removes/purges the core thin-edge.io packages like `tedge, tedge_mapper, and tedge_agent`.
> > It also removes/purges the thin-edge.io plugins like `tedge_apt_plugin, c8y_log_plugin, and c8y_configuration_plugin`.

Reviewer: ```suggestion
> Note: The uninstall script removes/purges the core thin-edge.io packages like `tedge, tedge_mapper, and tedge_agent` as well as thin-edge.io plugins like `tedge_apt_plugin, c8y_log_plugin, c8y_configuration_plugin` etc.
```

Author: fixed
